apps/web/src/features/athletes/components/athlete.form.tsx

```
@@ -0,0 +1,906 @@
+
+							return (
+								<div className="space-y-2">
+									<Label htmlFor="sales_setter_id">Sales Setter</Label>
+									<Popover
+										open={salesSetterOpen}
+										onOpenChange={setSalesSetterOpen}
+									>
+										<PopoverTrigger asChild>
+											<Button
+												variant="outline"
+												role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)